package.json

```diff
@@ -19,6 +19,9 @@
     "eslintrc.json",
     "index.js"
   ],
+  "engines": {
+    "node": ">=4.2.1"
+  },
   "devDependencies": {
```

Comment:
FYI: NodeJS v4.2.1 was the first officially declared v4 LTS version rather than just v4.x.x

More info at https://github.com/nodejs/LTS/